modin/pandas/test/test_groupby.py

```diff
@@ -1293,6 +1293,52 @@ def test_groupby_multiindex(groupby_kwargs):
     df_equals(md_grp.first(), pd_grp.first())
 
 
+@pytest.mark.parametrize("dropna", [(True), (False)])
+@pytest.mark.parametrize(
+    "groupby_kwargs",
+    [
+        pytest.param({"level": 1, "axis": 1}, id="level_idx_axis=1"),
+        pytest.param({"level": 1}, id="level_idx"),
+        pytest.param({"level": [1, "four"]}, id="level_idx+name"),
+        pytest.param({"by": "four"}, id="level_name"),
+        pytest.param({"by": ["one", "two"]}, id="level_name_multi_by"),
+        pytest.param({"by": ["item0", "one", "two"]}, id="col_name+level_name"),
+    ],
+)
+def test_groupby_with_kwarg_dropna(groupby_kwargs, dropna):
+    frame_data = np.random.randint(0, 100, size=(2 ** 6, 2 ** 4))
```

Comment:
```suggestion
    frame_data = np.random.randint(0, 100, size=(2 ** 6, 2 ** 4)).astype(float)
```
this could fix the tests